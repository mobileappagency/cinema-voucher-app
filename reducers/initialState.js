// @flow
import type { TvShowsStore } from './tvShowsReducer'

type ReduxState = {
  tvShows: TvShowsStore
}

const initialState: ReduxState = {
  tvShows: { genres: [], results: [] }
}

export default initialState
export type { ReduxState }
