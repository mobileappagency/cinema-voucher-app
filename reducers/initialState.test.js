// @flow
import initialState from './initialState'

describe('initialState', () => {
  it('matches current snapshot', () => {
    expect(initialState).toMatchSnapshot()
  })
})
