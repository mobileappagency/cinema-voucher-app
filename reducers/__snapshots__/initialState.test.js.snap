// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`initialState matches current snapshot 1`] = `
Object {
  "tvShows": Object {
    "genres": Array [],
    "results": Array [],
  },
}
`;
